recurly/base_client.py

```diff
@@ -13,10 +13,17 @@
 
 PORT = 443
 HOST = "v3.recurly.com"
-
 BINARY_TYPES = ["application/pdf"]
 
 
+def request_converter(value):
```

Comment:
There could be a better place to put this and test this. Maybe move all the `json.dumps` code out into it's own file and expose it as one function. I think this is fine for now.